### PR TITLE
5 of the seed files improved with some of the provided data and some …

### PR DESCRIPTION
**Pull request by fawadik**
…raw data created from the mockaroo.

Seed data improved in the half of the seed files as it was instructed.

### CODE REVIEW

File: src/server/constants/db-constants.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  TABLE_SUGGESTED_ROUTES: "suggested_routes",
+  TABLE_USER_ROLES: "user_roles",
+  TABLE_USERS: "users",
+  TABLE_VESSEL_REPORTS: "vessel_reports",
+  TABLE_VESSELS: "vessels",
+  TABLE_WAYPOINTS: "waypoints",
+  TABLE_FAVORITE_VESSELS: "favorite_vessels"
+};
+
+module.exports = constants;
```
Comment (zkwsk) [inline src/server/constants/db-constants.js]: Why are you adding a constants file when we agreed not to use it? Further, you are not referencing it anywhere.

**Review by zkwsk — CHANGES_REQUESTED** [review]

File: src/server/constants/db-constants.js (hunk 1 above)
Comment (fawadik) [reply src/server/constants/db-constants.js]: Oh.... i created it and then i forgot to delete it once we agreed upon not using it.

Comment (zkwsk) [reply src/server/constants/db-constants.js]: Allright. Make sure to always look at your file diff when you create a PR.

**Review by zkwsk — CHANGES_REQUESTED** [review]
You have merge conflicts. I've just merged quite a few PRs that affect the DB, so please test and double test that everything still works right. Delete and create your DB before running `npm run db:setup` just to make sure it will work for everyone.

**Review by lil2400 — APPROVED** [review]
All good.

**Review by zkwsk — APPROVED** [review]
Looks good 👍 Great work 🐘

**Pull request merged**